src/server/qtquick/wsurfaceitem.cpp

```diff
@@ -919,15 +919,7 @@ void WSurfaceItem::onSurfaceCommit()
 
     updateSurfaceState();
 
-    // Maybe the beforeRequestResizeSurfaceStateSeq is set by resizeSurfaceToItemSize,
-    // the resizeSurfaceToItemSize wants to resize the wl_surface to current size of WSurfaceitem,
-    // If change the WSurfaceItem's size at here, you will see the WSurfaceItem flash.
-    if (d->beforeRequestResizeSurfaceStateSeq < d->surface->handle()->handle()->current.seq) {
-        if (d->beforeRequestResizeSurfaceStateSeq != 0) {
-            Q_ASSERT(d->beforeRequestResizeSurfaceStateSeq == d->surface->handle()->handle()->current.seq - 1);
-            d->beforeRequestResizeSurfaceStateSeq = 0;
-        }
```

Comment:
```
  if (d->beforeRequestResizeSurfaceStateSeq <= d->surface->handle()->handle()->current.seq) {
        if (d->beforeRequestResizeSurfaceStateSeq != 0) {
            Q_ASSERT(d->beforeRequestResizeSurfaceStateSeq == d->surface->handle()->handle()->current.seq);
            d->beforeRequestResizeSurfaceStateSeq = 0;
        }
```

Reply:
beforeRequestResizeSurfaceStateSeq is `pending.seq`, it should equal current.seq in next commit

Reply:
beforeRequestResizeSurfaceStateSeq is updated without waiting for ack configure, can skip many seq nums